Client/TeamTalkClassic/TeamTalk.rc

```diff
@@ -306,7 +306,7 @@ STYLE DS_SETFONT | DS_FIXEDSYS | WS_CHILD | WS_VISIBLE
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
     CONTROL         "",IDC_RICHEDIT_CHANMESSAGES,"RICHEDIT20W",WS_BORDER | WS_VSCROLL | WS_TABSTOP | 0x1844,0,0,176,125
-    EDITTEXT        IDC_EDIT_CHANMESSAGE,0,126,176,14,ES_AUTOHSCROLL
+    EDITTEXT        IDC_EDIT_CHANMESSAGE,0,126,176,14,ES_MULTILINE,ES_AUTOHSCROLL

```

Comment:
Have you tested if Shift+Enter works for multiline?

Reply:
I've tested shift+enter on MP dialog so I think this work as well on this new multiline field.
For username and password translations, I don't know how to fix it :(, I must to add a loadtext maybe ?

Reply:
No, it's not working. I'll do more test of implementation.

Reply:
It probably works if you Copy/Paste and press Enter to have multiline.